core/entity.json

```diff
@@ -1,9 +1,9 @@
 {
-  "$id": "https://raw.githubusercontent.com/dcic/signature-commons-schema/master/core/entity.json",
+  "$id": "https://raw.githubusercontent.com/jeevangelista/signature-commons-schema/master/core/entity.json",
```

Comment:
These name changes are fine in perhaps your own branch to get the validation to use your own schemas, but to merge them into master they should be left alone.